src/sonic-daemon-base/sonic_daemon_base/daemon_base.py

```diff
@@ -28,20 +28,89 @@
 EEPROM_MODULE_NAME = 'eeprom'
 EEPROM_CLASS_NAME = 'board'
 
+EMPTY_NAMESPACE = ''
+ASIC_CONF_FILENAME = 'asic.conf'
+NAMESPACE_PREFIX = "asic"
+
+# TODO, to move this defenition to a common place
+INTERNAL_INTERFACE_PREFIX = "Ethernet-BP"
+
 #
 # Helper functions =============================================================
 #
 
-def db_connect(db_name):
+def db_connect(db_name, namespace=EMPTY_NAMESPACE):
     from swsscommon import swsscommon
     return swsscommon.DBConnector(db_name,
                                   REDIS_TIMEOUT_MSECS,
-                                  True)
+                                  True, namespace)
+
+def get_internal_interface_prefix():
+    return INTERNAL_INTERFACE_PREFIX
 
 #
 # Helper classes ===============================================================
 #
 
+class MultiAsic(object):
+    @staticmethod
+    def get_num_asics():
+        asic_conf_file_path = os.path.join(PLATFORM_ROOT_DOCKER, ASIC_CONF_FILENAME)
```

Comment:
We are creating APIs like this in several places. Can we reuse the APIs from sonice_device_utils and 
https://github.com/Azure/sonic-utilities/pull/999/

Reply:
I am in the middle of consolidating all this common functionality into a new sonic-py-common package, because I have seen too much of this type of thing happening lately. Please see https://github.com/Azure/sonic-buildimage/issues/4999

Reply:
cool. Some of these could be used on sonic-mgmt too. Wondering if these can be used in sonic-mgmt too.

Reply:
I guess we could try installing the new sonic-py-common package in the sonic-mgmt container, then it would be available to import from Pytests.